packages/c/calamares/package.yml

```diff
@@ -17,36 +17,36 @@ summary    : Distribution-independent installer framework
 description: |
     Calamares is a distribution-independent system installer, with an advanced partitioning feature for both manual and automated partitioning operations. Calamares is designed to be customizable by distribution maintainers without the need for cumbersome patching, thanks to third-party branding and external modules support.
 builddeps  :
-    - pkgconfig(Qt5Core)
-    - pkgconfig(Qt5Quick)
-    - pkgconfig(Qt5Svg)
-    - pkgconfig(Qt5UiTools)
-    - pkgconfig(Qt5WebEngineWidgets)
+    - pkgconfig(Qt6Core)
+    - pkgconfig(Qt6Quick)
+    - pkgconfig(Qt6Svg)
+    - pkgconfig(Qt6UiTools)
+    - pkgconfig(Qt6WebEngineWidgets)
     - pkgconfig(icu-i18n)
     - pkgconfig(libparted)
-    - pkgconfig(polkit-qt5-1)
+    - pkgconfig(polkit-qt6-1)
     - pkgconfig(pwquality)
     - pkgconfig(python3)
     - pkgconfig(yaml-cpp)
     - extra-cmake-modules
-    - kconfig-devel
-    - kcoreaddons-devel
-    - kcrash-devel
-    - kdbusaddons-devel
-    - ki18n-devel
-    - kpackage-devel
-    - kparts-devel
+    - kf6-kconfig-devel
+    - kf6-kcoreaddons-devel
+    - kf6-kcrash-devel
+    - kf6-kdbusaddons-devel
+    - kf6-ki18n-devel
+    - kf6-kpackage-devel
+    - kf6-kparts-devel
+    - kf6-kwidgetsaddons-devel
     - kpmcore-devel
-    - kwidgetsaddons-devel
     - libboost-devel
     - plasma-framework-devel
```

Comment:
Are you sure it still needs `plasma-framework-devel`? It doesn't link against it and the functions of that function have been moved elsewhere in Plasma 6. `plasma-framework-devel` is a Kf5/Qt5 lib.

Reply:
I think this one it does need, because it will fail to configure without it. At least, I had to install it when I went to build it while making a patch.

Reply:
Correction, it throws a warning about it not being present during configure.